pyramid_oereb/core/records/view_service.py

```diff
@@ -80,7 +80,7 @@ class ViewServiceRecord(object):
     """
 
     def __init__(self, reference_wms, layer_index, layer_opacity, default_language,
-                 srid, proxies=None, legends=None):
+                 srid, proxies=None, verify_certificate=True, legends=None):
```

Comment:
"verify_certificate" is inserted before "legends". Is that fine for every use of this class, or would it be better to have it at the end?

Can you update the documentation of the class?

Reply:
I moved it to the end, to reduce risks. 
Documentation is now also updated.